static_report/src/utils/store.ts

```diff
@@ -213,17 +214,15 @@ const getTableColumnsOnly = (rawData: ComparableReport) => {
       const base = baseColumns[key];
       const target = targetColumns[key];
       let mismatched = false;
+      const changeStatus = compareColumn(base, target);
 
-      if (!base) {
+      if (changeStatus === 'added') {
         added += 1;
         mismatched = true;
-      } else if (!target) {
+      } else if (changeStatus === 'removed') {
         deleted += 1;
         mismatched = true;
-      } else if (
-        base.name !== target.name ||
-        base.schema_type !== target.schema_type
-      ) {
+      } else if (changeStatus === 'implicit' || changeStatus === 'modified') {
```

Comment:
seems `compareColumn` only return `added`, `removed` and `implicit`

Reply:
But the changes status contains 'implicit' and 'modified', i think it would be ok to cover the both cases.